src/pybind/mgr/dashboard/frontend/src/app/shared/enum/cell-template.enum.ts

```diff
@@ -4,6 +4,16 @@ export enum CellTemplate {
   perSecond = 'perSecond',
   checkIcon = 'checkIcon',
   routerLink = 'routerLink',
+  // Display the cell with an executing state. The state can be set to the `cdExecuting`
+  // attribute of table rows.
+  // It supports an optional custom configuration:
+  // {
+  //   ...
+  //   cellTransformation: CellTemplate.executing,
+  //   customTemplateConfig: {
+  //     valueClass?: string;       // Cell value classes.
+  //     executingClass?: string;   // Executing state classes.
+  // }
```

Comment:
Is this dead code?

Reply:
They are example documents :) (to be consistent with the rest of file).